content/use-cases/tokenization/stablecoin-issuer.md

```diff
@@ -4,504 +4,173 @@ parent: tokenization.html
+The Authorized Trust Lines feature enables issuers to create tokens that can only be held by accounts that the issuer specifically authorizes. This feature only applies to tokens, not XRP.
 
-<!-- STYLE_OVERRIDE: feasible -->
+To use the Authorized Trust Lines feature, enable the `Require Auth` flag on your issuing account. While the setting is enabled, other accounts can only hold tokens you issue if you have authorized those accounts' trust lines to your issuing account.
 
-### Suspicious Activity Reporting
+See [Authorized Trust Lines](authorized-trust-lines.html).
 
-If a financial institution suspects that funds may be related to criminal activity, the institution must file a Suspicious Activity Report (SAR) with the appropriate regulatory authority. Failure to report suspicious activity may result in in penalties for the institution.
 
-See also:
+### Freeze a Trust Line
 
-- [Suspicious Activity Reporting Overview (USA FFIEC)](https://bsaaml.ffiec.gov/manual/RegulatoryRequirements/04_ep)
+If you issue tokens in the XRP Ledger, you can enable the _No Freeze_ setting to permanently limit your own ability to use the token freezing features of the XRP Ledger. (As a reminder, this only applies to issued tokens, not XRP.) 
 
-- [FATF Recommendation 16: Reporting of suspicious transactions and compliance](http://www.fatf-gafi.org/publications/fatfrecommendations/documents/fatf-recommendations.html)
+If you do not enable the _No Freeze_ setting, when an account shows suspicious activity or violates your institution's terms of use, you have the option of freezing the trust line while you resolve the issue.
 
-### Travel Rule
+See [Freezing Tokens](freezes.html).
 
-The Travel Rule is a Bank Secrecy Act (BSA) rule requiring funds-transmitting financial institutions to forward certain information to the next financial institution if the funds transmittal equals or exceeds the USD equivalent of $3,000. The following information must be included in the transmittal order:
 
-- The name of the transmittor,
-- The account number of the transmittor, if used,
-- The address of the transmittor,
-- The identity of the transmittor's financial institution,
-- The amount of the transmittal order,
-- The execution date of the transmittal order, and
-- The identity of the recipient's financial institution.
+### Global Freeze
 
-<!-- SPELLING_IGNORE: transmittor -->
+If you see signs of suspicious activity, you can enact a global freeze on your account to prevent users from sending your tokens to each other and trading your token in the decentralized exchange. 
```

Comment:
"suspicious activity" is vague here; there's all kinds of suspicious activity that doesn't justify a global freeze. Maybe it would be better as, "If you suspect that your own account or systems have been compromised," instead?